ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/HARouting.java

```diff
@@ -273,29 +276,37 @@ static void executeOrRouteQuery(
         pullQueryMetrics
             .ifPresent(queryExecutorMetrics -> queryExecutorMetrics.recordLocalRequests(1));
         pullPhysicalPlan.execute(locations, pullQueryQueue,  rowFactory);
+      } catch (StandbyFallbackException e) {
+        LOG.error("Standby Fallback: Error executing query {} locally at node {} with exception",
+            statement.getStatementText(), node, e.getCause());
+        throw e;
       } catch (Exception e) {
-        LOG.error("Error executing query {} locally at node {} with exception",
+        LOG.error("Error executing query {} locally at node {}",
             statement.getStatementText(), node, e.getCause());
         throw new KsqlException(
-            String.format("Error executing query %s locally at node %s",
-                          statement.getStatementText(), node),
+            String.format("Error executing query locally at node %s: %s", node.location(),
+                e.getMessage()),
             e
         );
       }
     } else {
       try {
         LOG.debug("Query {} routed to host {} at timestamp {}.",
-                  statement.getStatementText(), node.location(), System.currentTimeMillis());
+            statement.getStatementText(), node.location(), System.currentTimeMillis());
         pullQueryMetrics
             .ifPresent(queryExecutorMetrics -> queryExecutorMetrics.recordRemoteRequests(1));
         forwardTo(node, locations, statement, serviceContext, pullQueryQueue, rowFactory,
             outputSchema);
+      } catch (StandbyFallbackException e) {
+        LOG.error("Standby Fallback: Error forwarding query {} to node {} with exception {}",
```

Comment:
Do we want to log it as error really, since we are about to retry on the standby right?

Also if we intend to expose this to users, they may not really understand what `standby fallback` means. What about just "Failed to execute query {} on node {}, will try to switch to execute on the standby query state which may result in stale results"?